SocialCareCaseViewerApi/V1/Controllers/CaseStatusController.cs

```diff
@@ -81,6 +81,42 @@ public IActionResult CreateCaseStatus([FromBody] CreateCaseStatusRequest request
             }
         }
 
+        /// <summary>
+        /// Create case status answers
+        /// </summary>
+        /// <param name="request"></param>
+        /// <response code="201">Successfully created case status answers</response>
+        /// <response code="400">Invalid CreateCaseStatusAnswerRequest received</response>
+        /// <response code="404">Case status not found</response>
+        [HttpPost]
+        [Route("case-statuses/{id:long}/answers")]
+        public IActionResult CreateCaseStatusAnswers(CreateCaseStatusAnswerRequest request)
+        {
+            var validator = new CreateCaseStatusAnswerRequestValidator();
+            var validationResults = validator.Validate(request);
+
+            if (!validationResults.IsValid)
+            {
+                return BadRequest(validationResults.ToString());
+            }
+
+            try
+            {
+                _caseStatusesUseCase.ExecutePostCaseStatusAnswer(request);
+
+                return CreatedAtAction(nameof(CreateCaseStatusAnswers), "Successfully created case status answers.");
+            }
+            catch (Exception e) when (
+                e is WorkerNotFoundException)
+            {
+                return BadRequest(e.Message);
+            }
+            catch (CaseStatusDoesNotExistException e)
+            {
+                return NotFound(e.Message);
```

Comment:
My only suggestion is that I don't think a POST endpoint should be returning a 404, if the Case Status does not exist then the request was bad and a 400 seems appropriate.